astronomy/time_scales_body.hpp

```diff
@@ -403,6 +405,12 @@ constexpr Instant DateTimeAsUT1(DateTime const& ut1) {
   return FromUT1(TimeScale(ut1));
 }
 
+constexpr Instant JulianDateAsTT(JulianDate const& jd) {
+  return Instant() +
+         (jd.day() + (static_cast<double>(jd.fraction_numerator()) /
+                      static_cast<double>(jd.fraction_denominator()))) * Day;
+}
+
 // |Instant| date literals.
 
 #if (PRINCIPIA_COMPILER_CLANG || PRINCIPIA_COMPILER_CLANG_CL) && WE_LIKE_N3599
```

Comment:
N3599 doesn't look like it is going to happen, let us get rid of this.
If we want to enforce literal validity, we can resort to things like
```C++
define CONSTEXPR(expression) ([] { constexpr auto result = expression; return result; }())
```
```C++
CONSTEXPR("2000-01-01T11:59:27,816"_TAI)
```
or even
```C++
define TAI(string_literal) (CONSTEXPR(string_literal##_TAI))
```
```C++
TAI("2000-01-01T11:59:27,816")
```
Not really satisfying, but at least it's standard.

Reply:
Removed.